### PR TITLE
[AC-4670] add an __str__ method to RecommendationTag

### PR DESCRIPTION
**Pull request by yotammanor**

### CODE REVIEW

File: accelerator/models/recommendation_tag.py [hunk 1]
```diff
@@ -21,3 +20,6 @@ class RecommendationTag(AcceleratorModel):
     class Meta(AcceleratorModel.Meta):
         db_table = 'accelerator_recommendationtag'
         managed = settings.ACCELERATOR_MODELS_ARE_MANAGED
+
+    def __str__(self):
+        return "Recommendation Tag: {}".format(self.text)
```
Comment (pbx) [inline accelerator/models/recommendation_tag.py]: No `text` field that I can see.

Comment (yotammanor) [inline accelerator/models/recommendation_tag.py]: [line 18](https://github.com/masschallenge/django-accelerator/pull/4/files#diff-e807cf42c12cc5abdc4751a7db4539b3R18)

**Comment by yotammanor** [comment]
@pbx you're missing something :)

File: accelerator/models/recommendation_tag.py (hunk 1 above)
Comment (pbx) [inline accelerator/models/recommendation_tag.py]: Wow. Thanks for spelling it out, sorry for my reading impairment.

**Comment by yotammanor** [comment]
No worries @pbx, it kind of happened to me too when I was looking for the field to include in the __str__ :)

**Pull request merged**